### ISSUE TITLE
Add compatibility between smaller nether portals and crying portals

### ISSUE THREAD

**Issue by RandomGuy91** [ricksouth/serilum-mc-mods#1658]
ditto the title

**Comment by ricksouth** [comment]
This is already a thing. Which version are you on?

**Comment by RandomGuy91** [comment]
1.18.2, 2.3 of crying portals and 3.2 of smaller portals

**Comment by ricksouth** [comment]
This should work already, did you test it?

**Comment by RandomGuy91** [comment]
yeah, why would I not test it?

**Comment by RandomGuy91** [comment]
I am playing with a custom modpack I'm making tho. Here's my modlist: https://pastebin.com/KXAN6Ge3

**Comment by ricksouth** [comment]
I can create the portal on a fresh profile install on CurseForge on 1.18.2. Not sure what's causing the issue on your end.

![2023-03-06_14 17 28](https://user-images.githubusercontent.com/38502394/223121019-73218cd2-04d5-4a13-ba2c-3fbd9ab875cc.png)

**Comment by RandomGuy91** [comment]
here's a link to the video, as I say I have a lot of mods installed but if you don't know how to fix it, I definetly don't lol
https://drive.google.com/file/d/18M7kkiiZFnucMu-WJw5063-Sf5asBHnU/view?usp=sharing

**Comment by ricksouth** [comment]
How come the fire is blue, is that a resource pack?

**Comment by EightBitBBQ** [comment]
When you pause at 12 seconds, it says Soul Fire at the top!
I dont think portals can be lit with Soul Fire according to MC coding?

Sorry if i am budging in. But that could be a reason. Can you try to lit the portal with a Fire Charge for Example

**Comment by ricksouth** [comment]
Yep, that could be it! In that case, it should also not work with normal obsidian. Is that true?

**Comment by ricksouth** [comment]
I've just uploaded version 3.4 of Smaller Nether Portals which adds support for soul fire. @RandomGuy91 can you see if that fixes it?

https://www.curseforge.com/minecraft/mc-mods/smaller-nether-portals/files

**Comment by RandomGuy91** [comment]
I think Blue Flame Burning lights a soul fire on top of the crying obsidian. Unfortuantely, it still doesn't light when using soul fire...
